test/src/main/java/org/apache/accumulo/test/ample/metadata/TestAmple.java

```diff
@@ -213,6 +214,7 @@ protected Retry createUnknownRetry() {
         }
 
         @Override
+        @SuppressWarnings("deprecation")
```

Comment:
Why was this added?

Reply:
This was left over from a previous iteration while working on these changes. I removed it in b588423